fenix/app/nimbus.fml.yaml

```diff
@@ -326,6 +326,14 @@ features:
         type: List<OnboardingPanel>
         default: ["themes", "toolbar-placement", "sync", "tcp", "privacy-notice"]
 
+  glean:
+    description: "A feature that provides server-side configurations for Glean metrics (aka Server Knobs)."
+    variables:
+      glean-metric-configuration:
+        description: "A map of metric base-identifiers to booleans representing the state of the 'enabled' flag for that metric."
+        type: String
+        default: "{}"
```

Comment:
Could we use `Map<String, Boolean>` instead here? I'm thinking about the experimenter interface for experiment owners here: having an opaque string they type in, with not even JSON syntax to validate.

Reply:
@jhugman 
I talked with Travis a bit earlier today. The purpose of using the JSON string rather than something more structured is intentional to avoid too tight of coupling between Nimbus and Glean. This is a bit different since desktop has a Nimbus JSON type, but we are parsing and validating the JSON structure on the Glean side to ensure nothing unexpected happens.

That being said, if this is seen as a must-have I can make updates and then just convert the Nimbus value to a JSON string before we pass it to Glean.